scalafix-core/shared/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -251,11 +210,13 @@ object EscapeHatch {
       }
 
     private def extractRules(mods: List[Mod]): List[(String, Position)] = {
-      def process(rules: List[Term]) = rules.map {
+      def process(rules: List[Term]) = rules.flatMap {
```

Comment:
FYI `.collect` is a good fit here

Reply:
that's true... too late now. I can update it as part of a future PR